lab_7_llm/main.py

```diff
@@ -2,28 +2,24 @@
 Neural machine translation module.
```

Comment:
Checkout the files from 7 lab to the master

Reply:
Done!)